model/UsersData.ts

```diff
@@ -0,0 +1,151 @@
+import * as fsPromise from "fs";
+import { User } from "./User";
+import { File } from "./File";
+import { input } from "../utility/constants";
+
+export class UsersData {
+  public usersData: User[];
+  constructor() {
+    //Loading data from disk to in-memory
+    const data = fsPromise.readFileSync("./database/db.txt", "utf8");
+    let allUsers: any = data.split(",").slice(1);
+
+    allUsers = allUsers.map((element: any) => {
+      let bufferObj = Buffer.from(element, "base64");
+      return JSON.parse(bufferObj?.toString("utf8"));
+    });
+    this.usersData = allUsers;
+  }
+
+  public getUserData() {
+    return this.usersData;
+  }
+
+  public displayUserDetails(inputChoice:number,sortOrder:boolean) {
+    if(inputChoice == 1){
+      this.usersData = this.sortByName(this.usersData,sortOrder);
```

Comment:
While displaying data the sorting is done on the original array of users, which will reflect on storing the file. 
It should only be displayed in sorted manner, the original order (by roll number) should be retained.

Reply:
Done